src/components/Features/Features.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { FiFilter } from 'react-icons/fi';
+import { GrLanguage } from 'react-icons/gr';
+import { SiGotomeeting } from 'react-icons/si';
+import FeatureCard from './FeatureCard';
+
+const Features = () => {
+  return (
+    <section
+      data-testid="features-section"
+      className="bg-background dark:bg-gray-900"
+    >
+      <div className="py-8 px-4 mx-auto max-w-screen-xl sm:py-16 lg:px-6">
+        <div className="mb-8 lg:mb-16">
+          <h2
+            data-testid="features-title"
+            className="mb-4 text-4xl tracking-tight font-extrabold text-gray-900 dark:text-white"
+          >
+            Designed for your needs and kind
```

Comment:
Maybe writing 'designed to meet your needs.' instead

Reply:
Thanks sara, for the colors i used the background and secondary color from the palate, for the responsive i will update it en shaa allah, and for the title kind means kindness, because when you donate something it's because you kindness

Reply:
As mentioned below, the site is not only for donations therefore I'd go with what Sarah said, or something similar of your own liking.